src/components/v2/Layout/Layout.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+import Box from '@mui/material/Box';
+import { Sidebar } from './Sidebar';
+import { Header } from './Header';
+import { PageContainer } from './PageContainer';
+
+/* sidebar width */
+const drawerWidth = 180;
```

Comment:
Should this be defined as a theme constant and imported here?

Reply:
this variable is used just in scope of layout, but if you think that we should define it inside theme, i would move it